app/src/main/kotlin/com/wire/android/ui/home/conversations/details/participants/GroupConversationParticipantsViewModel.kt

```diff
@@ -76,23 +71,25 @@ open class GroupConversationParticipantsViewModel @Inject constructor(
         navigationManager.navigateBack()
     }
 
-    fun openProfile(participant: UIParticipant) = viewModelScope.launch {
-        if (participant.isSelf) navigateToSelfProfile()
-        else if (participant.isService && participant.botService != null) navigateToServiceProfile(participant.botService)
-        else navigateToOtherProfile(participant.id)
+    fun openProfile(participant: UIParticipant, direction: Direction? = null) = viewModelScope.launch {
+        if (participant.isSelf) {
+            navigateToSelfProfile()
+        } else if (participant.isService && participant.botService != null) {
+            navigateToServiceProfile(direction ?: ServiceDetailsScreenDestination(participant.botService, conversationId))
+        } else {
+            navigateToOtherProfile(direction ?: OtherUserProfileScreenDestination(participant.id, conversationId))
+        }
     }
 
-    private suspend fun navigateToSelfProfile() =
+    private suspend fun navigateToSelfProfile() {
         navigationManager.navigate(NavigationCommand(SelfUserProfileScreenDestination))
+    }
 
-    private suspend fun navigateToOtherProfile(id: UserId) =
-        navigationManager.navigate(
-            NavigationCommand(
-                OtherUserProfileScreenDestination(id, conversationId)
-            )
-        )
+    private suspend fun navigateToOtherProfile(direction: Direction) {
+        navigationManager.navigate(NavigationCommand(direction))
+    }
 
-    private suspend fun navigateToServiceProfile(botServiceId: BotService) {
-        navigationManager.navigate(NavigationCommand(ServiceDetailsScreenDestination(botServiceId, conversationId)))
+    private suspend fun navigateToServiceProfile(direction: Direction) {
+        navigationManager.navigate(NavigationCommand(direction))
```

Comment:
These two doesn't make sense anymore, I suggest to bring back the previous implementations